lib/questionnaires/model/src/questionnaire_enums.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:fhir/r4.dart';
+
+abstract class QuestionnaireResponseStatus {
+  static const completed = FhirCode.asConst('completed');
+  static const inProgress = FhirCode.asConst('in-progress');
+  static const amended = FhirCode.asConst('amended');
```

Comment:
In `develop`, we use snake case for naming the properties. This update might break the other projects that are using this package. Have you considered this?

Even if we proceed with snake case, we still need to update the other projects, so I think it's worth keeping the changes for consistency.

Reply:
@hanny-ph Thanks for the comment.

> In develop, we use snake case for naming the properties.

Sorry, in which part of `develop` are we using snake case for this? I actually had kept these in snake case before, but the linter kept giving warnings about these not being in camel case.

> This update might break the other projects that are using this package. Have you considered this?

Yes, I've considered this as well.
From my point of view, if we update the `fhir` package, regardless of using snake or camel case here, there's no way we can fully avoid breaking any projects depending on this package:
* The `QuestionnaireResponseStatus` enums, for example, are not coming anymore from the `fhir` package, but from this one.
* The FHIR Questionnaire/QuestionnaireResponse properties where these are being used are not instances of `QuestionnaireResponseStatus`, `QuestionnaireItemType`, etc, anymore, but `FhirCode`s instead.

Not sure if there's maybe some way to update the `fhir` package without incurring in breaking behavior on projects importing this. I think the best we can do is leave a `Breaking behavior` comment in the changelog at the most.

Please let me know if you have better ideas about this.

Reply:
> in which part of develop are we using snake case for this? 

Not in this repo, but in our private project.

> I think the best we can do is leave a Breaking behavior comment in the changelog at the most.

I see, in that case, I will resolve the discussion, thank you for the explanation.